app/Resources/assets/scss/main.scss

```diff
@@ -144,6 +144,24 @@ body#homepage {
 // Page: 'login'
 // --------------------------------------------------
 body#login {
+    #login-users-help p {
+        font-size: $font-size-base;
+        line-height: $line-height-base;
+
+        &:last-child {
+            margin-bottom: 0;
+        }
+
+        .label {
+            margin-right: 5px;
+        }
+
+        .console {
```

Comment:
The thing is that this "console" is very special in this page because I want to display it as a "soft console" not a "strong console" with a dark background. Here it is in action:

![login-console](https://cloud.githubusercontent.com/assets/73419/6869191/7f88087a-d492-11e4-87b1-fd59c740f278.png)